### ISSUE TITLE
Get Theme-Specified Tab Text Color for "disableColors" option

### ISSUE THREAD

**Issue by dfrankland** [dfrankland/hyperterm-tab-icons#12]
A configuration for disabling icons with different colors was added here:
https://github.com/dfrankland/hyperterm-tab-icons/pull/11

The default for that option simply turns all icons white, but it'd be nice to get the default theme color for tab icons.

**Comment by dfrankland** [comment]
If we can get the element that is the tab text then we should be able to do this:

``` javascript
const tabTextColor = window.getComputedStyle(tabTextElement).getPropertyValue('color');
```

**Comment by adccb** [comment]
Hey, I'd like to fix this. I'll do some digging and see if I can't find out more about the problem space. How does a PR sound before the end of the week?

**Comment by dfrankland** [comment]
@lcrossan, sounds good to me 👍  take your time.

**Comment by dfrankland** [comment]
This should be fixed with `hyper-tab-icons@1.1.4`, the relevant code commit block is here for anyone interested: https://github.com/dfrankland/hyper-tab-icons/blob/be8e341cb66437c6b0916ae18d0bb9e65fd854b9/index.js#L72-L90
